client/post-editor/editor-sidebar/feedback-header.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import EditorPostType from 'post-editor/editor-post-type';
+
+// TODO: Consider merging this with EditorSidebarHeader depending on final design
+function FeedbackSidebarHeader( { translate, closeFeedback } ) {
+	return (
+		<div className="editor-sidebar__header">
+			<EditorPostType isSettings />
+			<Gridicon icon="arrow-right" />
+			{ translate( 'Share Draft' ) }
+			<Button
+				compact borderless
+				className="editor-sidebar__back"
+				onClick={ closeFeedback }
+				title={ translate( 'Close feedback' ) }
```

Comment:
Thank you! It's good to know and will help me embrace our role in the translation effort.

I'll review the existing strings and update as needed.